manual/manual/refman/lex.etex

```diff
@@ -80,8 +80,8 @@ float-literal:
           [("e"||"E") ["+"||"-"] ("0"\ldots"9") { "0"\ldots"9"||"_" }]
         | ["-"] ("0x"||"0X")
           ("0"\ldots"9"||"A"\ldots"F"||"a"\ldots"f")
-          { "0"\ldots"9"||"A"\ldots"F"||"a"\ldots"f"||"_" }\\
```

Comment:
I disagree with this change, this `\\` was needed for the the latex version version of the manual to not overflow the corresponding latex box.